dashboard/app/views/pd/application/teacher1920_application_mailer/declined.html.haml

```diff
@@ -4,7 +4,7 @@
 
 %p
   Thank you for applying to
-  = @application.effective_regional_partner_name + "'s'"
+  = @application.effective_regional_partner_name + "'s"
```

Comment:
A regional partner who's name ends in s should just have an apostophe appended to indicate possession

`STEM Center of Kansas' program` for ex

Reply:
I actually think [both ways are correct](https://www.grammarly.com/blog/apostrophe/), but if we have a convention/preference for just using an apostrophe in such cases I can change it.